### PR TITLE
CASMPET-5977 - `csm_rbd_tool` and `cubs_tool` documentation

### PR DESCRIPTION
**Pull request by cdelatte-hpe**
# Description

Usage docs for csm_rbd_tool and cubs_tool.

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by bklei — APPROVED** [review]

**Review by bo-quan — APPROVED** [review]

**Review by rgross727 — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

File: operations/utility_storage/Cubs_tool_Usage.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Cubs_tool Usage
```
Comment (rustydb) [inline operations/utility_storage/Cubs_tool_Usage.md:1]:
```suggestion
# `cubs_tool` Usage
```

File: operations/utility_storage/Cubs_tool_Usage.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+## Introduction
+
+Cubs_tool is a python script developed as a second tier Ceph upgrade watching tool.  This was to better integrate the Ceph upgrade process with the upgrade workflow tooling.
```
Comment (rustydb) [inline operations/utility_storage/Cubs_tool_Usage.md:5]:
```suggestion
`cubs_tool` is a python script developed as a second tier Ceph upgrade watching tool.  This was to better integrate the Ceph upgrade process with the upgrade workflow tooling.
```

File: operations/utility_storage/Cubs_tool_Usage.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+## Glossary
+
+* **in_family** - is referring to an upgrade staying within the same major version of Ceph.  E.g, Any upgrade within the same CSM release will contain the same Major version of Ceph, but could have minor version bumps or patched containers.
```
Comment (rustydb) [inline operations/utility_storage/Cubs_tool_Usage.md:9]:
```suggestion
* **`in_family`** - is referring to an upgrade staying within the same major version of Ceph.  E.g, Any upgrade within the same CSM release will contain the same Major version of Ceph, but could have minor version bumps or patched containers.
```

File: operations/utility_storage/Cubs_tool_Usage.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  
+## Usage
+
+***IMPORTANT:*** The cubs_tool.py utility at this time will only work on `ncn-s00[1-3]`.  Please ensure you are only attempting to run this tool from one of those servers.
```
Comment (rustydb) [inline operations/utility_storage/Cubs_tool_Usage.md:13]:
```suggestion
***IMPORTANT:*** The `cubs_tool.py` utility at this time will only work on `ncn-s00[1-3]`.  Please ensure you are only attempting to run this tool from one of those servers.  
```

**Review by rustydb — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Review by rustydb — CHANGES_REQUESTED** [review]
Whitespace problems.

**Review by nrockershousen — DISMISSED** [review]
Please add entries for the new files in the operations README file.

**Review by denniswalker — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**